components/dashboard/src/index.css

```diff
@@ -9,39 +9,47 @@
 @tailwind utilities;
 
 @layer base {
+    html, body {
+        @apply h-full;
+    }
+    body {
+        @apply bg-white dark:bg-gitpod-black text-black dark:text-white;
+    }
     h1 {
-        @apply text-gray-900 font-bold;
+        @apply text-gray-900 dark:text-gray-50 font-bold;
         line-height: 64px;
         font-size: 48px;
     }
     h2 {
         @apply text-base text-gray-500;
     }
     h3 {
-        @apply text-2xl text-gray-800 leading-9 font-semibold;
+        @apply text-2xl text-gray-800 dark:text-gray-100 leading-9 font-semibold;
     }
     h4 {
-        @apply pb-2 text-sm font-semibold text-gray-600;
+        @apply pb-2 text-sm font-semibold text-gray-600 dark:text-gray-400;
     }
     p {
-        @apply text-sm text-gray-400;
+        @apply text-sm text-gray-400 dark:text-gray-600;
     }
 
     .btn-login {
-        @apply rounded-md border-none bg-gray-100 hover:bg-gray-200 text-gray-500;
+        @apply rounded-md border-none bg-gray-100 hover:bg-gray-200 text-gray-500 dark:text-gray-200 dark:bg-gray-800 dark:hover:bg-gray-600;
+    }
+    .dark .dark\:filter-invert {
+        @apply filter-invert;
     }
-
 }
 
 @layer components {
     button {
-        @apply cursor-pointer px-4 py-2 my-auto bg-green-600 hover:bg-green-700 text-gray-100 text-sm font-medium rounded-md focus:outline-none focus:ring transition ease-in-out;
+        @apply cursor-pointer px-4 py-2 my-auto bg-green-600 dark:bg-green-700 hover:bg-green-700 dark:hover:bg-green-600 text-gray-100 dark:text-green-100 text-sm font-medium rounded-md focus:outline-none focus:ring transition ease-in-out;
     }
     button.secondary {
         @apply bg-gray-100 hover:bg-gray-200 text-gray-500 hover:text-gray-600;
```

Comment:
**issue**: We are going to need a `bg-gray-700` and `text-gray-100` for the secondary button here as well as a `bg-gray-600` on hover for the dark theme.